serve.py

```diff
@@ -100,19 +115,25 @@ async def goto_dandiset(request, dataset):
         json_info = req.json()
         if json_info is not None:
             id = json_info['_id']
-            return response.redirect(f"https://gui.dandiarchive.org/#/dandiset/{id}")
+            url = f"https://gui.dandiarchive.org/#/dandiset/{id}"
+            if request.method == "HEAD":
+                return response.html(None, status=302, headers=make_header(url))
```

Comment:
And it count be be just response.redirect for GET? Strange that platform didn't support that

Reply:
not sure what you mean here.

Reply:
sorry -- phone typos made it even more unreadable than if it was just me ;)  next line in the file is `response.redirect(url)` so I wondered why it magically wouldn't be enough (unless `response` has no clue about .method of the request, and could not be customized with it)

Reply:
actually probably it doesn't even matter what the `method` is for redirection response - it will be for a client to follow to that url, and just issue HEAD to it.  So why just not `response.redirect(url)` in either GET or HEAD, that is my question ;)

Reply:
and
```
redirect(to, headers=None, status=302, content_type='text/html; charset=utf-8')
    Abort execution and cause a 302 redirect (by default).
    
    :param to: path or fully qualified URL to redirect to
    :param headers: optional dict of headers to include in the new request
    :param status: status code (int) of the new request, defaults to 302
    :param content_type: the content type (string) of the response
    :returns: the redirecting Response
```
so it is already 302